### ISSUE TITLE
Feature request: Add option to copy password to clipboard

### ISSUE THREAD

**Issue by MindTwister** [ejcx/passgo#1]
It would be nice if there was an option to copy the password to clipboard instead of showing it, either as a prompt or as a flag.

example 1:

    > passgo -copy news.ycombinator.com 
   Enter master password:
   Password has been copied to clipboard

example 2:

    > passgo news.ycombinator.com
    Enter master password:
    Copy password to clipboard [y/n]? y
    Password has been copied to clipboard.

If building in password copy does not make sense, another option might be to be able to pass the master password to stdin

example 3:

    > echo "no7 50 s3cr7" | passgo news.ycombinator.com | gocopy

Basically, I'd really like to make long complicated passwords, and I dont want to type them in.

Something like https://github.com/atotto/clipboard might come in handy

**Comment by ejcx** [comment]
Thanks for the report. This is definitely near the top of my to-do list.

**Comment by ejcx** [comment]
By the way @hsson implemented this in #4

**Comment by ejcx** [comment]
This was fixed in #14 . Woot!
